.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,20 @@
 ## What changed?
-<!-- Describe what has changed in this PR -->
+<!-- Describe what has changed in this PR. -->
+
 
 ## Why?
-<!-- Tell your future self why have you made these changes -->
+<!-- Tell your future self why have you made these changes. -->
+
 
 ## How did you test it?
-<!-- How have you verified this change? Tested locally? Added a unit test? Checked in staging env? -->
+- [ ] built
```

Comment:
```suggestion
- [ ] yolo
- [ ] built
```
(not a serious suggestion)

Reply:
"tested in production"
"my code doesn't need testing, it is flawless"
"checked by Chuck Norris"

There a bunch of good options.